broker/yarn.lock

```diff
@@ -9956,31 +9966,20 @@ fsevents@^1.0.0:
   languageName: node
   linkType: hard
 
-"mongoose-legacy-pluralize@npm:1.0.2":
-  version: 1.0.2
-  resolution: "mongoose-legacy-pluralize@npm:1.0.2"
-  peerDependencies:
-    mongoose: "*"
-  checksum: a1f86450b3540a1883cfb093001846806d14cd3a8b80f6681fdac13afcfd4a1f57875fb79009aceacf2e80e3ffd2d5ab7e289ba873c586958ce536eff270dc0a
-  languageName: node
-  linkType: hard
-
-"mongoose@npm:5.9.24":
-  version: 5.9.24
-  resolution: "mongoose@npm:5.9.24"
+"mongoose@npm:6.0.5":
+  version: 6.0.5
+  resolution: "mongoose@npm:6.0.5"
   dependencies:
-    bson: ^1.1.4
-    kareem: 2.3.1
-    mongodb: 3.5.9
-    mongoose-legacy-pluralize: 1.0.2
-    mpath: 0.7.0
-    mquery: 3.2.2
+    bson: ^4.2.2
+    kareem: 2.3.2
+    mongodb: 4.1.1
+    mpath: 0.8.4
```

Comment:
Here, `mpath` gets updated to 0.8.4 from 0.7.0 as it is a direct dependency of `mongoose`